src/QuiqChatClient.js

```diff
@@ -63,11 +70,19 @@ class QuiqChatClient {
 
   onError = (callback: (error: ?ApiError) => void): QuiqChatClient => {
     this.callbacks.onError = callback;
+    registerCallbacks({onError: callback});
```

Comment:
Nitpick/thought: Would it be better to give StubbornFetch callbacks within QuiqChatClient, which can in turn invoke the callback provided by the end user? I know it makes no difference functionality wise, it just feels like the communication with the "end client" should happen through `QuiqChatClient`